lighthouse-cli/chrome-launcher.ts

```diff
@@ -127,43 +126,39 @@ export class ChromeLauncher {
     }
 
     return Promise.resolve()
-      .then(() => {
-        const installations = (<any>chromeFinder)[process.platform]();
+        .then(() => {
+          const installations = (<any>chromeFinder)[process.platform]();
 
-        if (installations.length < 1) {
-          return Promise.reject(new Error('No Chrome Installations Found'));
-        } else if (installations.length === 1 || this.autoSelectChrome) {
-          return installations[0];
-        }
+          if (installations.length < 1) {
+            return Promise.reject(new Error('No Chrome Installations Found'));
+          } else if (installations.length === 1 || this.autoSelectChrome) {
+            return installations[0];
+          }
 
-        return ask('Choose a Chrome installation to use with Lighthouse', installations);
-      })
-      .then(execPath => this.spawn(execPath));
+          return ask('Choose a Chrome installation to use with Lighthouse', installations);
+        })
+        .then(execPath => this.spawn(execPath));
   }
 
   spawn(execPath: string): Promise<any[]> {
     return new Promise(resolve => {
-      if (this.chrome) {
-        log.log('ChromeLauncher', `Chrome already running with pid ${this.chrome.pid}.`);
-        return resolve(this.chrome.pid);
-      }
-
-      const chrome = spawn(
-        execPath,
-        this.flags(),
-        {
-          detached: true,
-          stdio: ['ignore', this.outFile, this.errFile]
-        }
-      );
-      this.chrome = chrome;
-
-      fs.writeFileSync(this.pidFile, chrome.pid.toString());
-
-      log.verbose('ChromeLauncher', `Chrome running with pid ${chrome.pid} on port ${this.port}.`);
-      resolve(chrome.pid);
-    })
-    .then(pid => Promise.all([pid, this.waitUntilReady()]));
+             if (this.chrome) {
```

Comment:
i think this is because it wants to block level indent inside of the new Promise cb and not the return indent level.

this is automatic,